haiku-apps/alivejournal/alivejournal-1.61.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Client for LiveJournal online journalling community"
+DESCRIPTION="AliveJournal is a revamped version of Simon Huet's original client."
+HOMEPAGE="https://github.com/grahams/alivejournal"
+COPYRIGHT="2000 Simon Huet
+	2010 Sean Graham"
+LICENSE="GNU GPL v2"
+REVISION="1"
+gitSrcRevision="e3d2c76f0645076f4e05ea942427e0f6e919948b"
+SOURCE_URI="https://github.com/grahams/alivejournal/archive/$gitSrcRevision.tar.gz"
+CHECKSUM_SHA256="8215d4ec005fd345c15ece8223f022cc6d434d940dc3c699a00c87c4e8b6ac49"
+SOURCE_FILENAME="alivejournal-$portVersion-$gitSrcRevision.tar.gz"
+SOURCE_DIR="alivejournal-$gitSrcRevision"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	alivejournal = $portVersion
+	app:AliveJournal = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	cmd:sed
+	"
+
+BUILD()
+{
+	make OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	install -d $appsDir $docDir
+	install -t $appsDir objects/AliveJournal
+	addAppDeskbarSymlink $appsDir/AliveJournal
+	install -t $docDir README
+	sed -n '/http://www.bebits.com/app/1920/q;p' $docDir/README
```

Comment:
What do you think of this change:
```diff
-	sed -n '/http://www.bebits.com/app/1920/q;p' $docDir/README
+	sed -i -n -e '/^------------$/q;p' $docDir/README
```

Reply:
Which parts of the README file would that command change/get rid of? There are so many of those dashes in the README file :(

Reply:
Well, as you probably figured it out, this sed command scans each line to see if the pattern matches. As soon as it finds a match, it just **q**uits. And before it quits, each line which does not match gets **p**rinted. And since we used the `-n`, sed just _eats_ the contents of the file, performing the actions in the commands provided. The result is that this sed command keeps everything from the first line to the line just before the very first match. Well, I'm sure you understood all this soon after you asked the question ;-)

Reply:
Oh, ok.